rocketpool-cli/node/commands.go

```diff
@@ -641,54 +641,6 @@ func RegisterCommands(app *cli.App, name string, aliases []string) {
 				},
 			},
 
-			{
-				Name:      "set-voting-delegate",
```

Comment:
can we leave these in, but simply have them print a message about houston with a link to the medium article?

Reply:
I'm not fond of the idea. Some ppl just run `rp node` or `rp pdao` to find a cmd when giving instructions. This would require users to execute the command to remember which one is the correct.

Reply:
We would also add descriptive messages to the output.

ie,
```
~$ rocketpool node

NAME:
   rocketpool node - Manage the node

USAGE:
   rocketpool node command [command options] [arguments...]

COMMANDS:
[...]
   set-voting-delegate, sv                       (DEPRECATED) Use `rocketpool pdao set-voting-delegate` instead
   clear-voting-delegate, cv                    (DEPRECATED) Use `rocketpool pdao set-voting-delegate` instead
[...]

$ rocketpool node set-voting-delegate patches.eth
This command has been replaced with `rocketpool pdao set-voting-delegate` in the Houston release. See [link to medium article] for more information
```


```